settings.yaml

```diff
@@ -1,40 +1,80 @@
-# PILATES SETTINGS
-s3_io: False
-region: sfbay
+#########################################################################################################
+
+############################
+#     PILATES SETTINGS     #
+############################
+
+# scenario defs
+region: austin
 scenario: base
 start_year: 2010
 end_year: 2012
-land_use_freq: 1  # zero turns off land use models completely
+land_use_freq: 1
 travel_model_freq: 1
+
+# simulation platforms (leave blank to turn off)
+land_use_model: urbansim
+travel_model: polaris
+activity_demand_model:
+
+# docker settings
+docker_images:
+  urbansim: mxndrwgrdnr/block_model_v2_pb
+  beam: dimaopen/beam:0.8.6.3
+  activitysim: mxndrwgrdnr/activitysim
+  polaris:
 docker_stdout: False
 pull_latest: False
 
-# BEAM SETTINGS
-travel_model_image: dimaopen/beam:0.8.6.3
-path_to_skims: pilates/beam/beam_output/10.activitySimODSkims.UrbanSim.TAZ.Full.csv.gz
-beam_config: sfbay/gemini/activitysim-base-from-60k-input.conf
-beam_plans: sfbay/gemini/activitysim-plans-base-2010-cut-60k/plans.csv.gz
-beam_local_input_folder: pilates/beam/production/
-beam_local_output_folder: pilates/beam/beam_output/
-skim_zone_source_id_col: objectid
+# misc pilates settings
+skims_fname: campo_skims.hdf5
+# skims_fname: 10.activitySimODSkims.UrbanSim.TAZ.Full.csv.gz
+
+#########################################################################################################
 
-# URBANSIM SETTINGS
-land_use_image: mxndrwgrdnr/block_model_v2_pb
+###############################
+#    SUB-PLATFORM SETTINGS    #
+###############################
+
+# URBANSIM
 region_to_region_id:
   sfbay: "06197001"
+  austin: "48197301"
 usim_local_data_folder: pilates/urbansim/data/
 usim_client_data_folder: /base/block_model_probaflow/data/
 usim_formattable_input_file_name: "custom_mpo_{region_id}_model_data.h5"
 usim_formattable_output_file_name: "model_data_{year}.h5"
+usim_formattable_command: "-r {0} -i {1} -y {2} -f {3}"
 
 
-# ACTIVITYSIM SETTINGS
-activity_demand_image: mxndrwgrdnr/activitysim
+# BEAM 
+beam_config: activitysim-base-from-60k-input.conf
+beam_local_input_folder: pilates/beam/production/
+beam_local_output_folder: pilates/beam/beam_output/
+beam_plans: activitysim-plans-base-2010-cut-60k/plans.csv.gz
+region_to_beam_subdir:
+  example: beamville 
+  sfbay: sf-light
+  austin:
```

Comment:
I was trying to move away from region -> subdir mapping because usually we cannot switch beam scenarios by pointing out a single subdir. Beam often uses a very flexible directory structure. Though we can use this region_to_beam_subdir mapping but we would probably need to change beam_local_input_folder, and beam_config can be a path (not a file name).

Reply:
That's fine. I don't think these settings are actually being used anywhere. I think I left them in the PR by mistake.